tests/test_modules/test_model_contracts/test_text_classification.py

```diff
@@ -90,6 +90,13 @@ def test_post_process(simple_text_config):
     # With a low threshold, the post-processing is not affecting the predicted class with a random
     # model.
     assert np.all([pred.model_output.preds == pred.postprocessed_output.preds for pred in res_low])
+    # The prediction of the last pipeline step should be the same as the postprocessed prediction.
+    assert np.all(
+        [
+            pred.postprocessed_output.preds == pred.postprocessing_steps[-1].output.preds
+            for pred in res_low
+        ]
+    )
```

Comment:
Small trick just to make your life easier in the future: `all()` (not `np.all()`) accepts a generator directly, so no need to create a list comprehension.
```suggestion
    assert all(
        pred.postprocessed_output.preds == pred.postprocessing_steps[-1].output.preds
        for pred in res_low
    )
```